spec/requests/v1/items_request_spec.rb

```diff
@@ -38,20 +38,41 @@
 
         items = JSON.parse(response.body, symbolize_names: true)
 
+        
+
         items[:data].each do |item|
-            expect(item[:id].to_i).to be_an(Integer)
+            expect(item[:id]).to be_an(String)
+            # check for type
 
-            expect(item[:attributes]).to have_key(:name)
```

Comment:
Taking Adbul's advice here. the fact that we are checking the values of the keys within `attribute` makes the `.to have_key` redundant, I will implement this for my unit tests for merchants as well.